gui/src/installer/view/editor/template/mod.rs

```diff
@@ -0,0 +1,73 @@
+pub mod custom;
+pub mod inheritance;
+
+use iced::{alignment, Alignment, Length};
+
+use liana_ui::{
+    color,
+    component::{
+        button, card,
+        text::{h3, p1_regular, p2_regular},
+    },
+    widget::*,
+};
+
+use crate::installer::context;
+use crate::installer::{message::Message, view::layout};
+
+pub fn choose_descriptor_template(progress: (usize, usize)) -> Element<'static, Message> {
+    layout(
+        progress,
+        None,
+        "Choose Wallet Type",
+        Column::new()
+            .max_width(800.0)
+            .align_items(Alignment::Start)
+            .push(Container::new(
+                p1_regular("What do you want your wallet for? Also consider this depends on the amount of funds you have, the more funds, higher the security should be. Not sure about the wallet type? We can help you.")
+                .style(color::GREY_3)
+                .horizontal_alignment(alignment::Horizontal::Left)
+            ).align_x(alignment::Horizontal::Left).width(Length::Fill))
+            .push(
+                card::simple(
+                    Row::new()
+                        .align_items(Alignment::Center)
+                        .push(
+                            Column::new()
+                                .align_items(Alignment::Start)
+                                .push(h3("Simple inheritance"))
+                                .push(p2_regular("Two keys required, one for yourself to spend and another for your heir.").style(color::GREY_3))
```

Comment:
We alternate between "heir", "relative" and "loved one". Should we use a single term to avoid any confusion?